Maple2.Model/Metadata/AiMetadata.cs

```diff
@@ -0,0 +1,572 @@
+﻿using System.Collections.Generic;
+using System.Numerics;
+using System.Text.Json.Serialization;
+using Maple2.Model.Enum;
+using Maple2.Model.Game.Event;
+
+namespace Maple2.Model.Metadata;
+
+public enum AiConditionTargetState {
+    grabTarget,
+    holdme
+}
+
+public enum AiConditionOp {
+    Equal,
+    Greater,
+    Less,
+    GreaterEqual,
+    LessEqual,
+}
+
+public enum NodeSummonOption {
+    none,
+    masterHP,
+    hitDamage,
+    linkHP
+}
+
+public enum NodeSummonMaster {
+    Master,
+    Slave,
+    None
+}
+
+public enum NodeAiTarget {
+    defaultTarget,
+    hostile,
+    friendly
+}
+
+public enum NodeTargetType {
+    rand,
+    near,
+    far,
+    mid,
+    nearAssociated,
+    rankAssociate,
+    hasadditional,
+    randassociated,
+    grabbeduser,
+    random
+}
+
+public enum NodeJumpType {
+    JumpA = 1,
+    JumpB = 2
+}
+
+public enum NodeRideType {
+    Slave
+}
+
+public enum NodeBuffType {
+    remove,
+    add
+}
+
+public enum NodePopupType {
+    talk,
+    cutin
+}
```

Comment:
Move these enums to `Maple2.Model/Enum` 
Perhaps to a Ai.cs

Reply:
We should also use proper casing here